tests/services/test_file_svc.py

```diff
@@ -1,7 +1,6 @@
 import os
 import pytest
 
-from app.utility.payload_encoder import xor_file
```

Comment:
Removed this because it was throwing an warning while testing and wasn't being used (dead import).